packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```diff
@@ -47,11 +47,7 @@
     color($input-text-color),
     $alpha: $input-disabled-alpha--weak
   );
-  $input-darkmode: if(
-    color.lightness(color($input-bg-color)) < 50%,
-    true,
-    false
-  );
+  $input-darkmode: is-color-dark($input-bg-color);
```

Comment:
I noticed a couple of differences after this update. The styles might need a bit more tweaking. Here are a couple samples:

First, it seems to create better contrast for yellow-20v:
![Screenshot 2023-05-15 at 4 35 26 PM](https://github.com/uswds/uswds/assets/93996430/580e2d43-bae2-4b83-81b6-fb40455c34ae)

But it creates worse contrast for green-40 or cyan-40:
![Screenshot 2023-05-15 at 4 26 09 PM](https://github.com/uswds/uswds/assets/93996430/56c7059f-8df6-4255-b1e6-09c19266005b)

Reply:
Confirming that I'm able to recreate this issue when updating `$theme-input-background-color:`

Reply:
Wanted to add this additional context for testing `$theme-input-background-color`.

> $theme-input-background-color does not actually set the background color, it adapts the colors of the component to a specified background color.

Originally in https://github.com/uswds/uswds/pull/4199#issuecomment-854112418